app/main.py

```diff
@@ -1 +1,20 @@
-# write your code here
+from app.cafe import Cafe
+
+from app.errors import (
+    VaccineError,
+    NotWearingMaskError
+)
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    masks_to_buy = 0
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except VaccineError:
+            return "All friends should be vaccinated"
```

Comment:
The current logic returns immediately if any friend is not vaccinated, which is correct. However, it might be clearer to specify which friend is not vaccinated if the function is extended in the future. Consider logging or handling specific friends in a more detailed manner if needed.